src/util/toggle-class.js

```diff
@@ -0,0 +1,38 @@
+//
+// helper to add/remove classes for IE9
```

Comment:
Currently this code is used by *all* browsers. also this is relevant to IE11 because SVG elements don't have classList there. Maybe add a comment explaining why this file even exists in the era of classList and link to issue #147